### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by brad-c7** [bitfocus/companion-module-aja-helo#7]
Is there such a variable available to use in tandem with Feedback? That way, when recording or streaming, you could have your start/record button change colors, based on the actual status of the Helo, not just a reaction to the button press internally.

**Comment by Caleb-Wishart** [comment]
This already exists as the feedbacks are a subset of the variables.

The feedbacks are also based on the state of the Helo, not just the button press :)
You can set the feedback to monitor for recording, idle or error.


If you still want the variables:
You can see the values that they can be in #4 
The variable names can be found in the [source code](https://github.com/bitfocus/companion-module-aja-helo/blob/master/src/variables.js#L6)

In short:
| Var name | purpose |
| :-: | :-: |
| recorder_status | Helo Enum name |
| recorder_status_value | Helo Enum value|
| stream_status | Helo Enum name |
| stream_status_value | Helo Enum value|

**Comment by Caleb-Wishart** [comment]
@brad-c7 ,
Just wanted to check in to make sure that you were able to use the provided feedback and variables correctly?

:)

**Comment by brad-c7** [comment]
Looks like the provided info is working as it should. Thanks so much for your assistance! :)

**Comment by Caleb-Wishart** [comment]
Awesome!

Let me know if you have any other queries or issues.

:)
